Source/r_plane.c

```diff
@@ -375,6 +378,9 @@ static void do_draw_plane(visplane_t *pl)
 
 	    // Vertical offset allows careful sky positioning.
 
+	    if (dc_texheight >= 200)
+	      dc_texturemid = s->rowoffset + 200*FRACUNIT;
+	    else
 	    dc_texturemid = s->rowoffset - 28*FRACUNIT;
```

Comment:
Do we still move the sky texture against the map author's choice?

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/483/commits/f04ed9607e3613334cd656ccb03674ba0aec5989